<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GuiNetworkLogin</class>
 <widget class="QDialog" name="GuiNetworkLogin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>287</width>
    <height>208</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="4">
    <widget class="QFrame" name="mp_frameLogin">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Sunken</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="mp_lService">
        <property name="text">
         <string>Account</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="mp_comboService"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="mp_lJid">
        <property name="text">
         <string>User</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="mp_leUser"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="mp_lPassword">
        <property name="text">
         <string>Password</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="mp_lePassword">
        <property name="echoMode">
         <enum>QLineEdit::Password</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QCheckBox" name="mp_cbRememberMe">
        <property name="text">
         <string>Remember user</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QCheckBox" name="mp_cbRememberPassword">
        <property name="text">
         <string>Remember password</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QCheckBox" name="mp_cbAutomaticConnection">
        <property name="text">
         <string>Automatic connection</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>106</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QPushButton" name="mp_pbLogin">
     <property name="text">
      <string>Login</string>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="mp_pbSave">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="1" column="3">
    <widget class="QPushButton" name="mp_pbClose">
     <property name="text">
      <string>Close</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>mp_comboService</tabstop>
  <tabstop>mp_leUser</tabstop>
  <tabstop>mp_lePassword</tabstop>
  <tabstop>mp_cbRememberMe</tabstop>
  <tabstop>mp_cbRememberPassword</tabstop>
  <tabstop>mp_cbAutomaticConnection</tabstop>
  <tabstop>mp_pbLogin</tabstop>
  <tabstop>mp_pbSave</tabstop>
  <tabstop>mp_pbClose</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
